plugin/cmd/kube-scheduler/app/configurator.go

```diff
@@ -194,3 +210,87 @@ func (sc schedulerConfigurator) Create() (*scheduler.Config, error) {
 
 	return sc.CreateFromConfig(*policy)
 }
+
+func (sc *schedulerConfigurator) SetupPolicyConfigMapEventHandlers(client clientset.Interface, informerFactory informers.SharedInformerFactory) {
+	// selector targets only the scheduler's policy ConfigMap.
+	selector := cache.NewListWatchFromClient(client.CoreV1().RESTClient(), "configmaps", sc.policyConfigMapNamespace, fields.OneTermEqualSelector(api.ObjectNameField, string(sc.policyConfigMap)))
+
+	sharedIndexInformer := informerFactory.InformerFor(&clientv1.ConfigMap{}, func(client clientset.Interface, resyncPeriod time.Duration) cache.SharedIndexInformer {
+		sharedIndexInformer := cache.NewSharedIndexInformer(
```

Comment:
BTW I just noticed that the API comment for policyConfigMapNamespace says that it defaults to kube-system (which makes sense) but it seems that's [not actually true](https://github.com/kubernetes/kubernetes/blob/b66be98174fb0257619c218f51246511d6160beb/plugin/cmd/kube-scheduler/app/options/options.go#L76)